### DISCUSSION TITLE
Configurar un question exec validate para que un campo de texto pueda ingresar fracciones

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#583] [2. Questions and Answers]
Hola a todos

El tema es el siguiente

Necesito que en un campo de tipo texto, pueda ingresar sólo números y "/" , para las fracciones.

Necesito confirmar si el code para que se me cumpla esa regla:

```        "validate" : {
            "src" : "function run()
			{const datos = context['responses#self'][0];
			// gets the value from the input
			if (!datos.match(/^[-\\w.%+A-Z]))
			return [{cmd: 'RESULT',result: false,value: 'No es un formato de fracciones'}];
			return [{ cmd: 'RESULT', result: true }];}", 
            "context" : "responses#self"
        }
```

Gracias =>

**Comment by acedward** [comment]
@NicolasPLG pudiste resolver este problema?
yo usaría el regex  `[\d/,. ]` o algo así.
Puedes probarlos en https://www.regextester.com/

**Reply by NicolasPLG** [reply]
hola @acedward lo probaré ahora con lo que me dices y te confirmo. =>
